docs/articles/documentation/test-api/executing-code-in-the-browser/dom-node-snapshot-members.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: docs
+title: DOM Node Snapshot Members
+permalink: /documentation/test-api/executing-code-in-the-browser/dom-node-snapshot-members.html
+---
+# DOM Node Snapshot Members
+
+*DOM node snapshots* are objects that represent the state of a particular document node on the server side.
+
+You can obtain a snapshot from the client by calling a [selector](selector-functions.md) function.
+
+Selector functions can return snapshots for DOM elements as well as document, fragment, text and comment nodes.
+
+This topic lists members exposed by DOM node snapshots.
+
+## Members Common across All Nodes
+
+Property | Type | Description
+------ | ---- | -----
+`nodeType` | Number | The type of the node. See [Node.nodeType](https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType).
+`textContent` | String | The text content of a node and its descendants. See [Node.textContent](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent).
+`childNodeCount` | Number | The number of child nodes.
+`hasChildNodes` | Boolean | `true` if this node has child nodes.
+`childElementCount` | Number | The number of child HTML elements.
+`hasChildElements` | Boolean | `true` if this node has child HTML elements.
+
+Method | Type | Description
+------ | ---- | -----
+`getParentNode` | Selector function | A selector that returns the parent node snapshot. See [Node.parentNode](https://developer.mozilla.org/en-US/docs/Web/API/Node/parentNode).
+`getChildNode(idx)` | Selector function | A selector that returns the snapshot of the child node at index `idx`. See [Node.childNodes](https://developer.mozilla.org/en-US/docs/Web/API/Node/childNodes).
+`getChildElement(idx)` | Selector function | A selector that returns the snapshot of the child element at index `idx`. See [ParentNode.children](https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/children).
+
+## Members Specific to Element Nodes
+
+Member | Type | Description
+------ | ---- | ----
+`tagName` | String | The name of the element. See [Element.tagName](https://developer.mozilla.org/en-US/docs/Web/API/Element/tagName).
+`visible` | Boolean | `true` if the element is visible.
+`focused` | Boolean | `true` if the element is focused.
+`attributes` | Object | Attributes of the element as `{ name: value, ... }`.
+`boundingClientRect` | Object | The size of an element and its position relative to the viewport. Contains `left`, `right`, `bottom`, `top`, `width` and `height` properties.
+`classNames` | Array of String | The list of classes of the element.
```

Comment:
I think it's rather `array of strings` if it's in a sentence or `Array of String` if we pretend it's code-ish